app/src/main/java/com/sun/gamevui/ui/adapter/GamesAdapter.kt

```diff
@@ -0,0 +1,47 @@
+package com.sun.gamevui.ui.adapter
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import com.sun.gamevui.R
+import com.sun.gamevui.base.BaseAdapter
+import com.sun.gamevui.base.BaseViewHolder
+import com.sun.gamevui.data.model.Game
+import com.sun.gamevui.databinding.ItemGameBinding
+
+class GamesAdapter(
+    private val onClickItem: (Game) -> Unit
```

Comment:
```suggestion
    private val onItemClick: (Game) -> Unit
```
để ý xem mấy hàm của Android đều viết theo kiểu v, onItemSelected các kiểu chứ có ai viết ngc lại đâu